README.md

```diff
@@ -1,38 +1,50 @@
 # Encryption in transit with mTLS
 
-Temporal supports Mutual Transport Layer Security (mTLS) as a way of encrypting network traffic between the services of a cluster and also between application processes and a Cluster. Self-signed or properly minted certificates can be used for mTLS. mTLS is set in Temporal's [TLS configuration](https://docs.temporal.io/references/configuration/#tls). The configuration includes two sections such that intra-Cluster and external traffic can be encrypted with different sets of certificates and settings:
+Temporal supports Mutual Transport Layer Security (mTLS) as a way of encrypting network traffic
+between the services of a cluster and also between application processes and a Cluster.
+Self-signed or properly minted certificates can be used for mTLS. mTLS is set in
+Temporal's [TLS configuration](https://docs.temporal.io/references/configuration/#tls).
+The configuration includes two sections such that intra-Cluster and external traffic can be
+encrypted with different sets of certificates and settings:
 
 - `internode:` Configuration for encrypting communication between nodes in the cluster.
 - `frontend:` Configuration for encrypting the Frontend's public endpoints.
-A customized configuration can be passed using either the [WithConfig](https://docs.temporal.io/references/server-options#withconfig) or [WithConfigLoader](https://docs.temporal.io/references/server-options#withconfig) Server options.
+A customized configuration can be passed using either the
+[WithConfig](https://docs.temporal.io/references/server-options#withconfig) or
+[WithConfigLoader](https://docs.temporal.io/references/server-options#withconfig) Server options.
 
 ## Prerequisites
 
 - Java JDK 17+
 - Docker
 - Azure CLI
 - openssl
+- jq
 
 `**Note:** In case the dev-container is used, all dependencies are already installed.`
 
 ## Get Started
 
-- Clone repository `git clone https://github.com/Aymalla/temporalio-mtls.git`
-- A keyvault need to be created and it's name replaced inside `deployment/certs/generate-test-certs-keyvault.sh` for variable `keyVault=<keyvault name>`
+- Clone repository `git clone https://github.com/Aymalla/temporalio-mtls.git`.
+- An Azure Key Vault need to be created to store certificates.
+The caller also needs `Key Vault Certificates Officer` and `Key Vault Secrets User` RBAC roles
+to create, import certificates and download private keys.
 - Login to your subscription `az login`
-- Generate certificates for mTLS using azure keyvault `make keyvault-certs`
-- Start temporal cluster `make start-temporal-cluster-mtls`
-- Start workflow worker `make start-worker`
-- Trigger new helloworld workflow instance `http://localhost:8000/workflow/start`
-- Access Temporal dashboard UI to check running history `http://localhost:8080/namespaces/default/workflows`
+- Generate certificates for mTLS using Azure Key Vault `make kv=<Key Vault Name> keyvault-certs`.
```

Comment:
make keyvault-certs  kv=<Key Vault Name>
The order of the parameter to be after the command is clearer for me